templates/server/src/main/resources/archetype-resources/src/main/java/__packageInPathFormat__/general/common/security/accesscontrol/ApplicationAccessControlConfig.java

```diff
@@ -0,0 +1,38 @@
+package ${package}.general.common.security.accesscontrol;
+
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.inject.Named;
+
+/**
+ * Example of {@link AccessControlConfig} that used for testing.
+ */
+@ApplicationScoped
```

Comment:
See the new dependency-injection guide. We want to propagate a programming approach that works for both spring and quarkus to simplify our guides and avoid "if spring then ... else if quarkus ...".
Feel free to discuss with all of us in a meeting.

Reply:
I kicked out JakartaEE for the moment so considered done.
However, it is not "ignored by java" but it might be ignored by spring.
IMHO it is never bad to make things explicit. In Quarkus it does make a difference and may be required e.g. for integration testing.